onnx/common/ir.h

```diff
@@ -946,10 +951,29 @@ friend struct Value;
     doc_string_ = std::move(doc_string);
   }
 
-  void addInitializer(Tensor initializer, std::string name) {
-    initializers_.push_back(std::move(initializer));
-    initializer_names_.push_back(std::move(name));
+  void addInitializer(Tensor& initializer) {
+    if (initializer.name().empty()) {
```

Comment:
For initializer comes from real world models, no it cannot be empty.
 
However, the reason why I made the condition here is it can be useful for creating a new initializer during version_converter or testing. In that case users don't need to worry about how to create a unique name. Also this is how existing addInitializerAndInput is doing.